deploy/controller/setup-controller.yaml

```diff
@@ -0,0 +1,1050 @@
+apiVersion: v1
```

Comment:
is it possible to have 3 files, one for CRD, one for RBAC, and one for deployment? `1050` lines in a single file seem heavy for me.

Reply:
Yeah, we can, but here we are just building the config/default which generates all these at a place. I would prefer to have it as a TODO as it required more modifications.